docs/reference/commandline/network_connect.md

```diff
@@ -16,14 +16,38 @@ parent = "smn_cli"
 
       --help=false       Print usage
 
-Connects a running container to a network. This enables instant communication with other containers belonging to the same network.
+Connects a running container to a network. You can connect a container by name
+or by ID. Once connected, the container can communicate with other containers in
+the same network.
 
+```bash
+$ docker network connect multi-host-network container1
 ```
-  $ docker network create -d overlay multi-host-network
-  $ docker run -d --name=container1 busybox top
-  $ docker network connect multi-host-network container1
+
+You can also use the `docker run --net` option to start a container and
```

Comment:
should we make this `docker run --net=<network-name>`, or similar,to prevent people from thinking `--net` doesn't require a value?